src/app/components/applications/SSV/MyAccount/components/ReactivateCluster/ReactivateCluster.tsx

```diff
@@ -40,18 +40,19 @@ const ReactivateCluster = () => {
   const timePeriodNotValid = customPeriod < 30;
   const process: SingleCluster = processStore.getProcess;
   const cluster = process.item;
+  const validatorCount = cluster.validator_count > 0 ? cluster.validator_count : 1;
```

Comment:
This can be simplified as:

```
const validatorCount = cluster.validator_count || 1;
```

Question why do you need so weird condition?

Reply:
done